services/NotificationAgent/Dockerfile

```diff
@@ -14,6 +14,5 @@ LABEL org.iplantc.de.notificationagent.git-ref="$git_commit" \
       org.iplantc.de.notificationagent.version="$version" \
       org.iplantc.de.buildenv.git-ref="$buildenv_git_commit"
 
-USER iplant
 ENTRYPOINT ["java", "-Dlogback.configurationFile=/etc/iplant/de/logging/notificationagent-logging.xml", "-cp", ".:notificationagent-standalone.jar", "notification_agent.core"]
```

Comment:
This line is depending on things being copied into `/home/iplant` and the workdir being `/home/iplant` -- probably those things will need to be changed to just use `/root` or `/`